dataCertPlots.py

```diff
@@ -366,16 +366,27 @@ def GetYAverage(hist,zeroSupress=True):
         histPLTLumi[run].Draw("histsame")
         histPCLumiB3p8[run].SetLineColor(802)
         histPCLumiB3p8[run].Draw("histsame")
-         
+
+	standInPC	= histPCLumiB3p8[run].Clone("PC")
+	standInHF	= histHFLumi[run].Clone("HF")
+	standInBCMF	= histBCMFLumi[run].Clone("BCMF")
+	standInPLT	= histPLTLumi[run].Clone("PLT")
+
+        mean7,meanError7=GetYAverage(standInPC,True)
+	mean8,meanError8=GetYAverage(standInHF,True)
+	mean9,meanError9=GetYAverage(standInBCMF,True)
+	mean10,meanError10=GetYAverage(standInPLT,True)
+
```

Comment:
So in this case we need to perform a different operation to determine the value that goes into the plot.  Since histPCLumiB3p8 is the instantaneous lumi, we must multiple the histogram by 23.31 seconds and then take the integral of the whole histogram.  

So lines like:
integrateLumiPC=histPCLumiB3p8[run].Integral() * 23.31;